apps/playlist/pkg/app/app.go

```diff
@@ -38,13 +38,27 @@ func New(cfg app.Config) (app.App, error) {
 		KubeConfig: cfg.KubeConfig,
 		InformerConfig: simple.AppInformerConfig{
 			ErrorHandler: func(ctx context.Context, err error) {
-				logging.DefaultLogger.With("error", err).Error("Informer processing error")
+				klog.ErrorS(err, "Informer processing error")
```

Comment:
QQ – why the switch to `klog`?

Reply:
primarily because there's already an adapter for grafana's logger, and grafana's logger can't be used directly because it's not a submodule. we need to add an adapter for the SDK's logger